beyond-essence/inventory/toggle-switches.md

```diff
@@ -89,16 +75,46 @@ To decide when to use a toggle switch, here are some clues:
 
 ## Copy writing
 
-* Short and direct.
-* Describe what the control will do when the switch is on.
+As toggle switches typically allow merchants to enable or disable features or modes that have a significant effect on their workflow and data might be lost when they deactivate a previously activated feature, make sure the label (and description, if applicable) are precise and make it easy for the merchant to understand the consequences of activating/deactivating the toggle.
+
+* Be short, precise and direct
+* Start the toggle label with a verb and describe what changing its default state will do, e.g. “Allow visitors to make purchases on your website.” (exception: “Website is live”)
+* If there is a static description text underneath the toggle, describe what will happen when the switch is activated.
+Start with “If this toggle is activated…”
+* If the description text only appears after the feature is toggled on, describe what will happen when the switch is turned off.
+Start with: “If you toggle off FEATURE…”
 * Don't use neutral or ambiguous phrases.
 * Avoid asking questions.
 
-```
-Track stock level
-```
+**DO**: "If this toggle is activated, there will be a cart displayed in the header of your website as well as an "Add to cart" button on each product.
```

Comment:
I'm not sure if everyone would be aware that these examples are for the description text and not for the toggle label?

Reply:
I'll add this in to be clear about it.